### ISSUE TITLE
Better support for markdown in `DiffComments`

### ISSUE THREAD

**Issue by pvinis** [react-native-community/upgrade-helper#48]
# Feature Request

This will allow us to add better comments on the diffs, without ending up with with a wall of text with no formatting.

**Comment by vinnyA3** [comment]
I can try taking a look @ this 👍

**Comment by lucasbento** [comment]
Hey @vinnyA3, we didn't focus on providing much content on this issue because I started working on it on the background, apologies for that.

Basically our current solution doesn't really fulfil our needs, we currently use [markdown-to-jsx](https://github.com/probablyup/markdown-to-jsx) which doesn't respect line breaks, that's why we had to do #47 and embed it on a comment. 

I'm working on migrating to [inline-mdx.macro](https://github.com/hamlim/inline-mdx.macro), it's taking some time because I have filled a few issues there where the library didn't work because of our structure, everything seems to be working fine currently so I'm planning to open a PR for it next week.

Thank you for the prompt response though!
